docs-src/typescript/chapter-durable-execution/add-sleep-for-one-minute.md

```diff
@@ -0,0 +1,43 @@
+---
+id: add-sleep-for-one-minute
+title: Add a call to sleep
+sidebar_label: Add sleep call
+description: Add a call to sleep for one minute to the beginning of the Workflow.
+tags:
+- timer
+- sleep
+- logger
+---
+
+In the following sample, we add a couple of logging statements and a Timer to the Workflow code to see how this affects the Event History.
+
+Use the `sleep()` API to cause the Workflow to sleep for a minute before the call to execute the Activity.
+
+Use the `console.log()` API to log from Workflows to avoid seeing repeated logs from the Replay of the Workflow code.
```

Comment:
Not sure what this means because Temporal's logging API won't result in duplicate messages
during Replay